src/ECC.DanceCup.Api.Infrastructure.Storage/DomainModel/Dbo/CategoryDbo.cs

```diff
@@ -0,0 +1,10 @@
+namespace ECC.DanceCup.Api.Infrastructure.Storage.DomainModel.Dbo;
+
+internal class CategoryDbo
+{
+    public long Id { get; set; }
+    
+    public required long TournamentId { get; set; }
+    
+    public required string Name{ get; set; }
```

Comment:
```suggestion
    public long TournamentId { get; set; }
    
    public string Name { get; set; } = string.Empty;
```

В DBO лучше делать не required, а значения по умолчанию, потому что мы можем не получить какое-то поле из бд, и в случае с required всё упадёт